packages/react-google-maps-api/src/utils/useAtMostOnce.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import { useEffect } from 'react'
+
+export const createUseAtMostOnce = (
```

Comment:
I doubt this is gonna work. The refs are like instance properties in classes. They are not shared. You would need a global variable which is bad. Let's not force this check upon the user.